LondonDataServices.IDecide.Portal.Server/appsettings.json

```diff
@@ -102,7 +102,19 @@
         "PhoneNumber": "07700901890",
         "RegisteredGpPractice": "E19201"
       }
-    ]
+    ],
```

Comment:
See previous comments

Reply:
If you add this to the core project, it will be able to see the internal classes in the acceptance test project:
```cs
		<InternalsVisibleTo Include="LondonDataServices.IDecide.Core.Tests.Acceptance" />
```